automation/test.sh

```diff
@@ -365,11 +366,17 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ (cnao|multus) ]]; then
     label_filter='(Multus,Networking,VMIlifecycle,Expose,Macvtap)'
   elif [[ $TARGET =~ sig-network ]]; then
+    label_filter='(sig-network)'
+    skip_tests=()
     # FIXME: https://github.com/kubevirt/kubevirt/issues/9158
     if [[ $TARGET =~ no-istio ]]; then
-      label_filter='(sig-network && !Istio)'
-    else
-      label_filter='(sig-network)'
+      skip_tests+=("Istio")
+    fi
+    if [[ $KUBEVIRT_WITH_MULTUS_V3 == "true" ]]; then
+      skip_tests+=("in-place-hotplug-NICs")
+    fi
+    if [[ ${#skip_tests[@]} -gt 0 ]]; then
+      label_filter="${label_filter} && (!($(IFS=,; echo "${skip_tests[*]}")))"
```

Comment:
Please use `add_to_label_filter` instead this logic
i.e on line 373 and on line 376 directly add to the `label_filter`
It will handle the case of empty list and non empty list

Reply:
Done